server/src/main/java/org/opensearch/search/aggregations/support/ValuesSourceAggregatorFactory.java

```diff
@@ -102,4 +102,16 @@ protected abstract Aggregator doCreateInternal(
     public String getStatsSubtype() {
         return config.valueSourceType().typeName();
     }
+
+    public String getField() {
+        return config.fieldContext().field();
+    }
```

Comment:
If we have the whole config exposed via `getConfig`, why do we need to have the `getField` method?

Reply:
Nice catch - I don't need both of these:

```
public String getAggregationName() {
        return name;
    }

    public ValuesSourceConfig getConfig() {
        return config;
    }
```

Will remove this in next iteration.